astronomer/providers/amazon/aws/example_dags/example_emr_job_flow_manual_steps.py

```diff
@@ -0,0 +1,104 @@
+"""Example DAG's for AWS EMR related operator and sensor"""
+
+import os
+from datetime import datetime, timedelta
+
+from airflow import DAG
+from airflow.models.baseoperator import chain
+from airflow.providers.amazon.aws.operators.emr import (
+    EmrAddStepsOperator,
+    EmrCreateJobFlowOperator,
+    EmrTerminateJobFlowOperator,
+)
+
+from astronomer.providers.amazon.aws.sensors.emr import EmrStepSensorAsync
+
+JOB_FLOW_ROLE = os.environ.get("EMR_JOB_FLOW_ROLE", "EMR_EC2_DefaultRole")
+SERVICE_ROLE = os.environ.get("EMR_SERVICE_ROLE", "EMR_DefaultRole")
+AWS_CONN_ID = os.environ.get("ASTRO_AWS_CONN_ID", "aws_default")
+EMR_CONN_ID = os.environ.get("ASTRO_EMR_CONN_ID", "emr_default")
```

Comment:
```suggestion
JOB_FLOW_ROLE = os.getenv("EMR_JOB_FLOW_ROLE", "EMR_EC2_DefaultRole")
SERVICE_ROLE = os.getenv("EMR_SERVICE_ROLE", "EMR_DefaultRole")
AWS_CONN_ID = os.getenv("ASTRO_AWS_CONN_ID", "aws_default")
EMR_CONN_ID = os.getenv("ASTRO_EMR_CONN_ID", "emr_default")
```

Since, we are doing a get anyway to return default value, we could also use getenv. The implementation is same behind the scene by being a wrapper, just saves up some characters :)